tests/click_tests/test_autolock.py

```diff
@@ -92,17 +91,15 @@ def test_autolock_interrupts_signing(device_handler: "BackgroundDeviceHandler"):
         script_type=messages.OutputScriptType.PAYTOADDRESS,
     )
 
-    device_handler.run(
-        btc.sign_tx, "Bitcoin", [inp1], [out1], prev_txes=TX_CACHE_MAINNET
-    )
+    device_handler.run_and_wait_for_ui(btc.sign_tx, "Bitcoin", [inp1], [out1], prev_txes=TX_CACHE_MAINNET)  # type: ignore
 
-    layout = debug.wait_layout()
-    assert "1MJ2tj2ThBE62zXbBYA5ZaN3fdve5CPAz1" in layout.get_content().replace(" ", "")
-
-    debug.click(buttons.OK, wait=True)
+    assert (
+        "1MJ2tj2ThBE62zXbBYA5ZaN3fdve5CPAz1"
+        in debug.wait_layout().text_content().replace(" ", "")
+    )
 
     layout = debug.click(buttons.OK, wait=True)
-    assert "Total amount: 0.0039 BTC" in layout.get_content()
+    assert "0.0038 BTC" in layout.text_content()
```

Comment:
It is because the first screen which we assert says - you are sending 0.38, and only a later screen says - you are sending 0,39 including 0.01 fee. This transaction-flow change is probably already in `master`, it is not rebased - my guess?